cmd/cstor-volume-mgmt/volume/volume.go

```diff
@@ -107,11 +107,11 @@ func extractReplicaStatusFromJSON(str string) (*apis.CVStatus, error) {
 func CreateIstgtConf(cStorVolume *apis.CStorVolume) []byte {
 
 	var buffer bytes.Buffer
-	buffer.WriteString(`# Global section
+	_, _ = buffer.WriteString(`# Global section
```

Comment:
This looks so tough to maintain.
What are the alternative ways?
Can we make use of go templating?

Reply:
Converted into go template.